platform/feature-api/api/src/pod-api/tests/uri.test.ts

```diff
@@ -0,0 +1,24 @@
+import { createUUID, isPolypodUri, PolyUri } from "../uri";
+
+describe("createUUID tests", () => {
+    it("should generate an UUID in the required format", () => {
+        expect(createUUID()).toMatch(/^\w{8}-\w{4}-4\w{3}-\w{4}-\w{12}$/);
+    });
+
+    it("should generate different subsequent UUIDs", () => {
+        let lastUUID = createUUID();
+        for (let i = 0; i < 100; i++) {
+            const thisUUID = createUUID();
+            expect(thisUUID).not.toEqual(lastUUID);
+            lastUUID = thisUUID;
+        }
+    });
+});
```

Comment:
as we're using the `uuid` library now, these can go... it's like we don't trust the lib enough :>

Reply:
`createUUID` is a wrapper, and can use that implementation, or not. This way we ensure that this will keep working (and will reveal crypto-insecure RNGs, as it did here)